config/daemonconfig.go

```diff
@@ -61,8 +61,9 @@ type DaemonConfig struct {
 }
 
 type MirrorConfig struct {
-	Host    string            `json:"host,omitempty"`
-	Headers map[string]string `json:"headers,omitempty"`
+	Host        string            `json:"host,omitempty"`
+	Headers     map[string]string `json:"headers,omitempty"`
+	AuthThrough bool              `json:"auth_through"`
```

Comment:
Should also add `omitempty` to json tag. Then the "auth_through" won't appear in json file if it is never set of be false.